docs/topics/mta-cli-analyze.adoc

```diff
@@ -25,27 +25,70 @@ Usage:
+      --json-output                       Create analysis and dependency
+                                          output as JSON.
+      --list-sources                      List rules for available
+                                          migration sources.
+      --list-targets                      List rules for available
+                                          migration targets.
+  -l, --label-selector (string)           Run rules based on specified label
+                                          selector expression.
+      --maven-settings (string)           Path to the custom maven
+                                          settings file to use.
+      --overwrite                         Overwrite output directory.
+      --skip-static-report                Do not generate the static report.
+  -m, --mode (string)                     Analysis mode, must be one of
+                                          `full` or `source-only`
+                                          (default: `full`).
+      --no-proxy (string)                 Proxy-excluded URLs
+                                          (relevant only with proxy).
+  -o, --output (string)                   Path to the directory for analysis
+                                          output.
+      --overwrite                         Overwrite output directory.
+      --rules (stringArray)               Filename or directory containing
+                                          rule files.
+      --skip-static-report                Do not generate the static report.
+  -s, --source (string)                   Source technology to consider
+                                          for analysis.
+                                          To specify multiple sources,
+                                          repeat the parameter:
+                                          `--source <source_1>
+                                          --source <source_2>` etc.
+  -t, --target (string)                   Target technology to consider
+                                          for analysis.
+                                          To specify multiple targets,
+                                          repeat the parameter:
+                                          `--target <target_1>
+                                          --target <target_2>` etc.
+
 
 Global Flags:
-      --log-level uint32   log level (default 4)
-      --no-cleanup         do not cleanup temporary resources
+      --log-level uint32                  Log level (default: 4).
+      --no-cleanup                        Do not cleanup temporary resources.
 ----
+
+[NOTE]
+====
+The list of flags above does not include the `--bulk` flag because the flag is only offered as part of a Developer Preview feature. That feature is described in xref:mta-cli-run-multiple-apps_{context}[Support for providing a single report when analyzing multiple applications on the CLI].
```

Comment:
```suggestion
The list of flags above does not include the `--bulk` flag because this flag is only offered as part of a Developer Preview feature. That feature is described in xref:mta-cli-run-multiple-apps_{context}[Support for providing a single report when analyzing multiple applications on the CLI].
```